packages/sources/src/components/image/image.stories.ts

```diff
@@ -0,0 +1,78 @@
+import { bindTemplate, StorybookParameters } from '../../stories-helpers';
+
+import { imageArgsMapper, imageArgTypes } from './image.args';
+import { Image } from './image.models';
+
+export interface ImageParameters<TemplateFnReturnType> {
+  imageTemplate: (imageProperties: Image) => TemplateFnReturnType;
+}
+
+export function storiesOfImage<TemplateFnReturnType>(
+  {
+    module: mainModule,
+    storiesOf,
+    readme
+  }: StorybookParameters,
+  {
+    imageTemplate: imageTemplate
```

Comment:
`imageTemplate: imageTemplate` is dubbelop, ik denk overblijfsel van refactoring/copy paste?

Reply:
Die heb ik over het hoofd gezien, is waarschijnlijk door hernoemen zo gekomen. Heb de dubbeling weggehaald